CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,194 @@
+organizers may take any action they deem appropriate, subject to correction 
+and reversal by the lead organizer. These include, but are not limited to:
+
+* A verbal warning (either in private or in public).
+* Eviction from an event with no refund (if applicable), with conditional 
+  right of return or without right of return.
+* A temporary ban of any length, with or without right of appeal, from any or
+  all community spaces.
+* A permanent ban, with or without right of appeal, from any or all community
+  spaces.
+* Temporary revocation of community organizer status for any length of time, 
+  with or without right of appeal.
+* Permanent revocation of community organizer status with or without right of 
+  appeal.
+* Removal of contributions to Pext from any Pext-associated repositories, with
+  or without right of appeal.
+* Notification of law enforcement, or other appropriate authorities, along with
+  any evidence of wrongdoing provided to the community organizers.
+
+## Reporting guidelines
+
+If you are subject to, or witness, unacceptable behaviour, or have any other
+concerns, please notify a community organizer as soon as possible. The current
+list of community organizers is:
+
+* Sylvia van Os (sylvia@hackerchick.me) (lead organizer)
+
+When reporting, please include the following information, to the best of your
+knowledge and ability:
+
+* When and where the unacceptable behaviour occurred.
+* Who engaged in unacceptable behaviour.
+* Which unacceptable behaviour(s) happened.
+* Any evidence for your report (such as screencaps, logs, etc).
+* Who (besides yourself) was affected by this behaviour, if applicable.
+* Whether you want any additional actions taken by the community organizers to 
+  ensure that your safety and well-being remain intact.
+
+Community organizers can assist local authorities and connect you
+with help or providing a safe environment insofar as they are able. In the 
+context of in-person events, organizers can also provide escorts as desired by 
+the person experiencing distress.
+
+## Addressing grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of
+Conduct, you should notify the community organizers with a concise
+description of your grievance. You should include the following information:
+
+* What report of unacceptable behaviour you are responding to.
+* Why you believe specifically that you did not engage in this behaviour, or
+  why you believe the enforcement actions taken against you are
+  disproportionate.
+* Any evidence you can provide for your grievance.
+* What actions you wish to be taken to address your grievance if found genuine.
+
+If you have been temporarily, or permanently, banned without right of appeal,
+any grievances filed with community organizers regarding the incident which
+caused the ban will be ignored.
+
+## Scope
+
+We expect all community participants (contributors, paid or otherwise; sponsors;
```

Comment:
"paid or otherwise" does not need to be stated, and the rest of the semicolons can be changed to commas.